environments/stg/stg.tfvars

```diff
@@ -1432,6 +1432,19 @@ frontends = [
       },
     ]
   },
+  {
+    name           = "sptribs-frontend"
+    custom_domain  = "sptribs-frontend.aat.platform.hmcts.net"
```

Comment:
Need to add the DNS value first >  https://github.com/hmcts/azure-public-dns/blob/master/environments/staging/aat.yml

Reply:
https://github.com/hmcts/azure-public-dns/pull/1038/files